OpenRA.Game/Graphics/Viewport.cs

```diff
@@ -74,6 +82,23 @@ public float Zoom
 		public static int TicksSinceLastMove = 0;
 		public static int2 LastMousePos;
 
+		float ClosestTo(float[] collection, float target)
+		{
+			var closest_value = collection.First();
+			var subtract_result = Math.Abs(closest_value - target);
```

Comment:
Can you change those to camelCase please? https://github.com/OpenRA/OpenRA/wiki/Coding-Standard